src/vs/workbench/services/languageRuntime/common/languageRuntimeIPyWidgetClient.ts

```diff
@@ -68,16 +68,14 @@ export class IPyWidgetClientInstance extends Disposable {
 		}));
 
 		// Forward messages from the runtime client to the webview.
-		this._register(_client.onDidReceiveData(data => {
-			this._logService.trace('RECV comm_msg:', data);
+		this._register(_client.onDidReceiveData(event => {
+			const data = event.data;
+			this._logService.trace(`RECV comm_msg: ${JSON.stringify(data)}`);
 
 			switch (data.method) {
```

Comment:
Any reason this is a `switch()` with a fallthrough case instead of an `if` statement?

Reply:
No reason. I thought there would be more `method` types that would require special handling.